addons/binding/org.openhab.binding.neeo/src/main/java/org/openhab/binding/neeo/internal/models/NeeoRecipe.java

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.neeo.internal.models;
+
+/**
+ * The model representing an Neeo Recipe (serialize/deserialize json use only).
+ *
+ * @author Tim Roberts - Initial contribution
+ */
+public class NeeoRecipe {
+
+    /** The recipe key */
+    private final String key;
+
+    /** The type of recipe (generally launch/poweroff) */
+    private final String type;
+
+    /** The name of the recipe */
+    private final String name;
+
+    /** Whether the recipe is enabled */
+    private final boolean enabled;
+
+    /** ?? whether the recipe is dirty ?? */
+    private final boolean dirty;
+
+    // May be used in the future...
+    // private final NeeoStep[] steps;
```

Comment:
The value is that I know what's is possible - quickly - without having to bring up all the tools I use to investigate what's possible (I've already done it once, documented - don't see the need to do it again if I decide to use them in the future).  Note: this already worked for me since I was able to add macros to the binding quickly.